bullet_train/docs/billing/stripe.md

```diff
@@ -42,96 +89,209 @@ Note this is different than how many Rails engines ask you to install migrations
+#### 4.2. Configure Stripe API Keys Locally
 
 Edit `config/application.yml` and add your new Stripe secret key to the file:
 
 ```yaml
-STRIPE_SECRET_KEY: sk_0CJw2Iu5wwIKXUDdqphrt2zFZyOCH
+STRIPE_SECRET_KEY: sk_test_aBunchOfRandomCharacters
 ```
 
-### 4.3. Populate Stripe with Locally Configured Products
+#### 4.3. Populate Stripe with Locally Configured Products
 
 Before you can use Stripe Checkout or Stripe Billing's customer portal, your locally configured products will have to be created on Stripe as well. To accomplish this, you can have all locally defined products automatically created on Stripe via API by running the following:
 
 ```
 rake billing:stripe:populate_products_in_stripe
 ```
 
-### 4.4. Add Additional Environment Variables
+#### 4.4. Add Additional Environment Variables
 
-The script in the previous step will output some additional environment variables you need to copy into `config/application.yml`.
 
+The script from the previous step will output some additional environment variables you need to copy into `config/application.yml`. It will look similar to the following:
+
+```
+OK, put the following in `config/application.yml` or wherever you configure your environment values:
+
+STRIPE_PRODUCT_BASIC_MONTHLY_2025_PRICE_ID: price_1QpbQSKKAAAAAAAAAAAAAAAA
+STRIPE_PRODUCT_BASIC_ANNUAL_2025_PRICE_ID: price_1QpbQSKKBBBBBBBBBBBBBBBB
+STRIPE_PRODUCT_PRO_ANNUAL_2025_PRICE_ID: price_1QpbQSKKCCCCCCCCCCCCCCCC
+```
 
-## 5. Wire Up Webhooks
+### 5. Wire Up Webhooks
 
 Basic subscription creation will work without receiving and processing Stripe's webhooks. However, advanced payment workflows like SCA payments and customer portal cancelations and plan changes require receiving webhooks and processing them.
 
-### 5.1. Ensure HTTP Tunneling is Enabled
+#### 5.1. Ensure HTTP Tunneling is Enabled
 
 Although Stripe provides free tooling for receiving webhooks in your local environment, the officially supported mechanism for doing so in Bullet Train is using [HTTP Tunneling with ngrok](/docs/tunneling.md). This is because we provide support for many types of webhooks across different platforms and packages, so we already need to have ngrok in play.
 
 Ensure you've completed the steps from [HTTP Tunneling with ngrok](/docs/tunneling.md), including updating `BASE_URL` in `config/application.yml` and restarting your Rails server.
 
-### 5.2. Enable Stripe Webhooks
+#### 5.2. Enable Stripe Webhooks
 
- - Visit [https://dashboard.stripe.com/test/webhooks/create](https://dashboard.stripe.com/test/webhooks/create).
- - Use the default "add an endpoint" form.
- - Set "endpoint URL" to `https://YOUR-SUBDOMAIN.ngrok.io/webhooks/incoming/stripe_webhooks`.
- - Under "select events to listen to" choose "select all events" and click "add events".
- - Finalize the creation of the endpoint by clicking "add endpoint".
+  - Visit [https://dashboard.stripe.com/test/workbench/webhooks](https://dashboard.stripe.com/test/workbench/webhooks).
```

Comment:
For me `https://dashboard.stripe.com/test/workbench/webhooks` isn't a valid address and it redirects to `https://dashboard.stripe.com/test/dashboard`. I can access the webhook overview screen at `https://dashboard.stripe.com/test/webhooks`.